### ISSUE TITLE
items not loaded in the same index like stored

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Neseek77/openmeetings#1218]
```
when you load the whiteboard-objects the sort is not the same for all
participants.
```

Original issue reported on code.google.com by `seba.wag...@gmail.com` on 23 Apr 2010 at 1:08
- Blocked on: #1216
- Merged into: #1217
